### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by DanRyanIrish**
This PR updates CHANGELOG with changes from PRs merged since 1.0 release.

### CODE REVIEW

**Comment by DanRyanIrish** [comment]
@Cadair, are these appropriate CHANGELOG entries for the post 1.0 merged PRs?  I copied from the SunPy CHANGELOG but if there's any formatting I've overlooked let me know.

Also, should the title of the CHANGELOG be ```1.1``` rather than ```1.0.1```?

**Comment by Cadair** [comment]
> Also, should the title of the CHANGELOG be 1.1 rather than 1.0.1?

yes

**Comment by DanRyanIrish** [comment]
Like this @Cadair?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,13 +1,24 @@
-1.0.1 (Unreleased)
+1.1 (Unreleased)
 ==================
```
Comment (Cadair) [inline CHANGELOG.rst]: underline is now too long.

Comment (DanRyanIrish) [reply CHANGELOG.rst]: Pesky underlining!!  Should now be fixed (with a sneaky force push ;) )

**Comment by DanRyanIrish** [comment]
Thanks @Cadair

**Pull request merged**